examples/standalone/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "standalone",
+  "version": "0.0.0",
+  "description": "Standalone pdf-react example",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "start": "./../../node_modules/.bin/babel-node ./example.jsx"
```

Comment:
What do you think if we get rid of the `.jsx` for a more standard `.js`.
Not sure why I did it this way, but it's odd